specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-02-15-preview/notebook.json

```diff
@@ -0,0 +1,461 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NotebookWorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted. The regenerate auth token operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../preview/2022-02-15-preview/cosmos-db.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/notebookWorkspaces/{notebookWorkspaceName}/start": {
+      "post": {
+        "tags": [
+          "NotebookWorkspacesResource"
+        ],
+        "operationId": "NotebookWorkspaces_Start",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "CosmosDBNotebookWorkspaceStart": {
+            "$ref": "./examples/CosmosDBNotebookWorkspaceStart.json"
+          }
+        },
+        "description": "Starts the notebook workspace",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NotebookWorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted. The start operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../preview/2022-02-15-preview/cosmos-db.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NotebookWorkspaceCreateUpdateParameters": {
+      "description": "Parameters to create a notebook workspace resource",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../preview/2022-02-15-preview/cosmos-db.json#/definitions/ARMProxyResource"
```

Comment:
>../../preview/2022-02-15-preview/cosmos-db.json#/definitions/ARMProxyResource [](http://example.com/codeflow?start=19&length=77)

Consider using Resource from common-types v2 or v3 types.json (depending on whether you want/need systemData).

Reply:
These schemas are the same as the previous ones, there is no change here. Can we take this as an action item so that it gets fixed in the next PR?